tests/raw_to_standard/icone_test.py

```diff
@@ -0,0 +1,218 @@
+        <radar devID="1724" intervalEnd="2020-08-21T15:45:00Z" latitude="37.2012256" longitude="-84.1573747" numReads="38" avgSpeed="69.21" stDevSpeed="6.9488" />
+        <radar devID="1724" intervalEnd="2020-08-21T15:50:00Z" latitude="37.2012256" longitude="-84.1573747" numReads="62" avgSpeed="64.11" stDevSpeed="9.0186" />
+        <radar devID="1724" intervalEnd="2020-08-21T15:55:00Z" latitude="37.2012256" longitude="-84.1573747" numReads="17" avgSpeed="66.91" stDevSpeed="5.3667" />
+        </sensor>
+        <sensor type="iCone" id="SB 5 - MP 47.5" latitude="37.2066700" longitude="-84.1691290">
+        <radar devID="1735" intervalEnd="2020-08-21T15:45:00Z" latitude="37.2066724" longitude="-84.1691283" numReads="70" avgSpeed="67.00" stDevSpeed="6.2133" />
+        <radar devID="1735" intervalEnd="2020-08-21T15:50:00Z" latitude="37.2066724" longitude="-84.1691283" numReads="42" avgSpeed="64.05" stDevSpeed="7.6332" />
+        <radar devID="1735" intervalEnd="2020-08-21T15:55:00Z" latitude="37.2066724" longitude="-84.1691283" numReads="45" avgSpeed="62.17" stDevSpeed="6.7055" />
+        </sensor>
+        <sensor type="iCone" id="SB 6 - MP 48.5" latitude="37.2193130" longitude="-84.2046600" />
+        <sensor type="iCone" id="SB 7 - MP 49.5" latitude="37.2299854" longitude="-84.2221508">
+        <radar devID="1719" intervalEnd="2020-08-21T15:40:00Z" numReads="19" avgSpeed="67.24" stDevSpeed="4.9229" />
+        <radar devID="1719" intervalEnd="2020-08-21T15:45:00Z" numReads="38" avgSpeed="65.00" stDevSpeed="10.7934" />
+        <radar devID="1719" intervalEnd="2020-08-21T15:50:00Z" numReads="62" avgSpeed="62.82" stDevSpeed="9.7647" />
+        <radar devID="1719" intervalEnd="2020-08-21T15:55:00Z" numReads="41" avgSpeed="59.57" stDevSpeed="8.8250" />
+        </sensor>
+        <sensor type="iCone" id="SB 8 - MP 50.5" latitude="37.2378880" longitude="-84.2359180" />
+        <display type="PCMS" id="I-75 SB - MP 50" latitude="37.2339700" longitude="-84.2290798">
+        <message verified="2020-08-21T15:42:27Z" text=" ROADWORK / 4 MILES / AHEAD // 19 MILES / OF WORK / MP 48-29" />
+        <message verified="2020-08-21T15:52:27Z" text=" ROADWORK / 4 MILES / AHEAD // 19 MILES / OF WORK / MP 48-29" />
+        </display>
+        <display type="PCMS" id="I-75 SB - MP 46" latitude="37.2059920" longitude="-84.1672690">
+        <message verified="2020-08-21T15:48:32Z" latitude="37.2060070" longitude="-84.1673050" text=" ROADWORK / NEXT / 5 MILES // 19 MILES / OF WORK / MP 48-29" />
+        </display>
+    </incident>
+    </incidents>"""
+    actual_standard = json.loads(json.dumps(icone.generate_standard_messages_from_string(
+        xml_string)))
+    for i in actual_standard:
+        del i['rtdh_timestamp']
+    expected = icone_test_expected_results.test_generate_standard_messages_from_string_expected
+    for i in expected:
```

Comment:
Should we just remove the rtdh_timestamp property from the known JSON rather than load it, loop, and remove?

Reply:
My intent was to make the expected perfectly accurate, but for the comparison I could not find a convenient way to mock the timestamp correctly (it kept having only a few decimal places be off even though I was mocking or using time machine libraries, very strange)
I would like to leave the expected the same, as it is the actual expected, and later I might be able to fix the timestamp mocking/comparison

Reply:
That makes sense, can you comment in the test with this explanation so we know what's going on?